api/src/main/java/org/motechproject/nms/api/web/UserController.java

```diff
@@ -70,16 +70,12 @@
     @ResponseStatus(HttpStatus.OK)
     public void setUserLanguageLocationCode(@PathVariable String serviceName,
                                             @RequestBody LanguageRequest languageRequest) {
-        String callingNumber = languageRequest.getCallingNumber();
-        String callId = languageRequest.getCallId();
-        String languageLocationCode = languageRequest.getLanguageLocationCode();
+        Long callingNumber = languageRequest.getCallingNumber();
+        Long callId = languageRequest.getCallId();
+        Integer languageLocationCode = languageRequest.getLanguageLocationCode();
```

Comment:
Ticket #79 is about converting languageLocationCode and defaultLanguageLocation code to Strings.  Should we do that here or in a different commit?